### PR TITLE
Fix the pod name for comDetails

### PR DESCRIPTION
**Pull request by liornoy**
Modifies the function for creating comDetails,
so the pod name will be more accurate.

### CODE REVIEW

File: endpointslices/endpointslices.go [hunk 1]
```diff
@@ -200,6 +201,24 @@ func getContainerName(portNum int, pods []corev1.Pod) (string, error) {
 	return res, nil
 }
 
+func getPodName(pod *corev1.Pod) string {
+	var res string
+	name := pod.OwnerReferences[0].Name
```
[the remark this answers is not in the document]
Comment (liornoy) [reply endpointslices/endpointslices.go:206]: would you agree that we will return empty string in that case?

[the remark this answers is not in the document]
Comment (liornoy) [reply endpointslices/endpointslices.go:206]: after a second look, it looks like the case where pods don't have OwnerReferences, they append the node name, so I'll do the same treatment like with Kind=Node

[the remark this answers is not in the document]
Comment (liornoy) [reply endpointslices/endpointslices.go]: I added this use case plus more others, verifying it against unit tests. please re-review :)

**Comment by sabinaaledort** [comment]
Fixed in https://github.com/openshift-kni/commatrix/pull/13

**Pull request closed without merging**